sema/check/sinks/csv_sink.py

```
@@ -0,0 +1,23 @@
+# your_submodule/sinks/csv_sink.py
+
+import csv
+from typing import List
+from ..testing.base import TestBase
+
+
+def write_csv(results: List[TestBase], output_file: str):
```

Ruff

Missing return type annotation for public function `write_csv`

Add return type annotation: `None`

(ANN201)